<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QObject {
	border: 1px solid grey;
	border-radius: 8px;
	margin: 4px;
	padding: 5px;
	font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stage">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_peer">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(98, 160, 234);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgba(96, 101, 101, 130);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left_bkt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_bkt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CE">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplay">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Build Graph  --------------------------------&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	background-color: rgb(222, 221, 218);
	margin: 4px
}

QPushButton:hover:!pressed {
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>711</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	padding: 5px;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>value &quot;X&quot; up to</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="formX">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	padding: 5px;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>value &quot;X&quot; from</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="upToX">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	padding: 5px;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>value &quot;Y&quot; from</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="formY">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	padding: 5px;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>value &quot;Y&quot; up to</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="upToY">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>result_show</zorder>
   <zorder>pushButton_ln</zorder>
   <zorder>pushButton_sin</zorder>
   <zorder>pushButton_stage</zorder>
   <zorder>pushButton_tan</zorder>
   <zorder>pushButton_acos</zorder>
   <zorder>pushButton_sqrt</zorder>
   <zorder>pushButton_cos</zorder>
   <zorder>pushButton_asin</zorder>
   <zorder>pushButton_log</zorder>
   <zorder>pushButton_atan</zorder>
   <zorder>pushButton_dot</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_peer</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_left_bkt</zorder>
   <zorder>pushButton_right_bkt</zorder>
   <zorder>pushButton_mod</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_divide</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_CE</zorder>
   <zorder>pushButton_multiplay</zorder>
   <zorder>pushButton_graph</zorder>
   <zorder>pushButton_X</zorder>
   <zorder>horizontalFrame</zorder>
   <zorder>label_3</zorder>
   <zorder>formX</zorder>
   <zorder>upToX</zorder>
   <zorder>label_5</zorder>
   <zorder>formY</zorder>
   <zorder>label_6</zorder>
   <zorder>upToY</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
